010_Intro/10_Installing_ES.asciidoc

```diff
@@ -110,22 +81,41 @@ like your own name, to stop ((("clusters", "changing default name")))your nodes
 the same network with the same name!
 
 You can do this by editing the `elasticsearch.yml` file in the `config/`
-directory and then restarting Elasticsearch.  When Elasticsearch is running in
-the foreground, you can stop it by pressing Ctrl-C; otherwise, you can shut
-it down with the `shutdown` API:
+directory and then restarting Elasticsearch.  
+
+When Elasticsearch is running in the foreground, you can stop it by pressing Ctrl-C.
 
+[[sense]]
+==== Installing Sense
+Sense is a https://www.elastic.co/guide/en/kibana/current/index.html[Kibana]
+app ((("Sense console", "Kibana app"))) that provides an interactive
+console for submitting requests to Elasticsearch directly from your browser.
+Many of the code examples in the online version of this book include a View in Sense link. When
+clicked, it opens up a working example of the code in the Sense console.
+You do not have to install Sense, but it will make this book much more
+interactive by allowing you to  experiment with the code samples on your local
+Elasticsearch cluster.
+
+((("Sense", "downloading and installing"))) To install and run Sense:
+
+. Run the following command in the Kibana directory to download and install the Sense app:
++
 [source,sh]
 --------------------------------------------------
-curl -XPOST 'http://localhost:9200/_shutdown'
+./bin/kibana plugin --install elastic/sense <1>
 --------------------------------------------------
+<1> Windows: `bin\kibana.bat plugin --install elastic/sense`.
++
+NOTE: You can download Sense from https://download.elastic.co/elastic/sense/sense-latest.tar.gz
```

Comment:
Should we also link to the manual installation docs for this?  https://www.elastic.co/guide/en/sense/current/installing.html#manual_download